internal/resource/tenancy.go

```diff
@@ -78,3 +79,52 @@ func DefaultNamespacedTenancy() *pbresource.Tenancy {
 		PeerName: "local",
 	}
 }
+
+// WildCardTenancyFor returns a valid tenancy with tenancy units set to wildcard for
+// the passed in scope.
+func WildCardTenacyFor(scope Scope) *pbresource.Tenancy {
```

Comment:
Other uses of 'wildcard' in the repo keeps the casing as `Wildcard` instead of `WildCard`. We may want to stick with one variant for consistency.

Reply:
Also there is a typo in the `Tenancy` part of the name.